pkg/api/routes.go

```diff
@@ -387,7 +388,8 @@ func (rh *RouteHandler) UpdateManifest(w http.ResponseWriter, r *http.Request) {
 	}
 
 	mediaType := r.Header.Get("Content-Type")
-	if mediaType != ispec.MediaTypeImageManifest {
```

Comment:
A helper like "isSuportedMediaType(mediaType) bool" would be useful here and above.